test/Microsoft.ML.Functional.Tests/ONNX.cs

```diff
@@ -51,15 +58,9 @@ public void SaveOnnxModelLoadAndScoreFastTree()
             var onnxEstimator = mlContext.Transforms.ApplyOnnxModel(modelPath);
             var onnxModel = onnxEstimator.Fit(data);
 
-            // TODO #2980: ONNX outputs don't match the outputs of the model, so we must hand-correct this for now.
```

Comment:
>puts don't match the outputs o [](start = 35, length = 30)

Can we keep a comment about this somewhere in the test? This is By Design, but nonetheless confusing.